File: RebeliaApp.Web/Dto/InfinityService/Request/FriendlyGameResultRequest.cs
using System;
using System.Collections.Generic;
using RebeliaApp.Web.Dto.Shared;
using static RebeliaApp.Web.Model.Enums;

namespace RebeliaApp.Web.Dto.InfinityService.Request
{
    public class FriendlyGameResultRequest
    {
        public List<PlayerResult> PlayerList { get; set; }
        public int? WinnerID { get; set; }
        public BattleResult BattleResult { get; set; }
        public DateTime Date { get; set; }
        public int SystemID { get; set; }
        public int ScenarioID { get; set; }
        public int Rounds { get; set; }
        public int PointFormat { get; set; }

    }
}

File: RebeliaApp.Web/ClientApp/src/model/Request/FriendlyGameResultRequest.ts
import { PlayerGameScore } from '../Shared/PlayerGameScore';
import { BattleResult } from '../Shared/Enums';


export class FriendlyGameResultRequest {
  date: Date;
  playerList: PlayerGameScore[];
  winnerID: number | null;
  pointFormat: number;
  rounds: number;
  systemID: number;
  scenarioID: number;

}
